loopback-sdk-angular-cli.travis.yml

```diff
@@ -0,0 +1,22 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : loopback-sdk-angular-cli
+# Source Repo         :https://github.com/strongloop/loopback-sdk-angular-cli.git
+# Travis Job Link     : https://travis-ci.com/github/genisysram/loopback-sdk-angular-cli/jobs/469731227
+# Created travis.yml  : No
+# Maintainer          : <ramakrishna.s@genisys-group.com>,<sethp@us.ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+
+
+sudo: false
+arch:
+    - amd64
+    - ppc64le
+language: node_js
+node_js:
+  - "8"
```

Comment:
Add the comment mentioning the reason for node 6

Reply:
@gururajrkatti 
nodejs is failing in master repo(amd 64) , and i didn't get any handy  solution to fix it , so i have removed it from the yml file.(only kept the latest version)
pls check  the  https://travis-ci.com/github/genisysram/loopback-sdk-angular-cli/jobs/465805819 .